packages/ember-toucan-core/src/components/form/file-input/list.hbs

```diff
@@ -1,4 +1,4 @@
-<ul class="m-0 list-none space-y-1 p-0" data-files ...attributes>
+<ul class="m-0 list-none space-y-1.5 p-0" data-files ...attributes>
```

Comment:
This matches the Figma file closer with `6px` of spacing rather than `4px` between items